src/loaders/configure/configure.js

```diff
@@ -0,0 +1,37 @@
+import { setAPI } from '../api/api'
+import { addToNREUM, gosCDN, gosNREUMInitializedAgents } from '../../common/window/nreum'
+import { setConfiguration, setInfo, setLoaderConfig, setRuntime } from '../../common/config/config'
+import { activateFeatures, activatedFeatures } from '../../common/util/feature-flags'
+import { isBrowserScope, isWorkerScope } from '../../common/util/global-scope'
+
+export function configure(agentIdentifier, opts = {}, loaderType) {
+    let { init, info, loader_config, runtime = {loaderType}, exposed = true } = opts
+    const nr = gosCDN()
+    let api = {}
+    if (!info) {
```

Comment:
This is where the decision is made to set the API to the window, or to the caller (class)

info is defined as an option, only in NPM. so if it is empty, we can assume the configs are on the page as is expected with CDN builds